.circleci/organise_doc.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const fs = require('fs-extra');
+const args = require('minimist')(process.argv.slice(2));
+
+// get SDK name and version from package files
+// they are used by JSDoc to generate the output directories
+const PACKAGE_JSON = require('../package.json');
+const PACKAGE_VERSION = PACKAGE_JSON.version;
+const PACKAGE_NAME = PACKAGE_JSON.name;
+
+// get the output directory
+const JSDOC_OUTPUT_DIR = require('../jsdoc.json').opts.destination;
```

Comment:
Again, this is nitpicky, so it's not worth changing this unless you're changing other things, but writing lines like this can sometimes make things difficult to debug (e.g. in the case of an uncaught exception thrown due to trying to treat `undefined` as an object). It's better to write like the following, so that stack traces point to the exact statement that caused the error:

```javascript
const jsdocJSON = require('../jsdoc.json');
const jsdocOpts = jsdoc.opts;
const jsdocOutputDir = jsdocOpts.destination;
```

Reply:
FWIW, I routinely ignore this suggestion in my own code, but it does improve debuggability, so I'm passing it on.